### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by eastermukora**
Created a PR template to fit issue #401 and extended to cover basic scenarios in contribution.

Issue #401 - Create a Pull request template

## Type of Change
Documentation Update

##Checklist
- [ ] I have tested the changes locally, and they work as expected
- [ ] The code follows the project's coding standards and conventions
- [ ] No query or DB changes made.
- [ ] I have updated some of the relevant documentation in contributing.md
- [ ] I have mentioned the person responsible for reviewing this PR.

##Reviewers
@RVA2869

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md
Comment (Ainali) [inline PULL_REQUEST_TEMPLATE.md:26]: We don't have a local database (yet).
```suggestion
- [ ] Any **query changes** have been verified and are working correctly.
```

**Comment by Ainali** [comment]
Thanks for working on this. In general, I think this is good, but there are a few things I think we should clean up before merging.

- The checkboxes in CONTRIBUTING are better as regular bullet points, as one will not really check anything there.
- I think we can skip the reviewer section and just use the GitHub functionality. Also, the contributor may not know who should review.

**Comment by eastermukora** [comment]
That is right, I'd also like help reviewing contribution.md in such a way that it's more direct or concise.

**Review by eastermukora — COMMENTED** [review]
Edited the files for changes recommended

**Review by Ainali — APPROVED** [review]
This is good, thanks!

**Review by Ainali — CHANGES_REQUESTED** [review]

File: PULL_REQUEST_TEMPLATE.md
Comment (Ainali) [inline PULL_REQUEST_TEMPLATE.md:29]: This check box can go too, we don't need the contributor to know who should review.

**Review by eastermukora — COMMENTED** [review]
removed the 'mention code reviewer check box'

**Review by Ainali — APPROVED** [review]

**Pull request merged**
